### ISSUE TITLE
[feature request] Adding words from Built-in Word List to  Skritter vocabulary

### ISSUE THREAD

**Issue by R0leeX** [cschiller/zhongwen#112]
We can already add words to skritter vocabulary. But using the Built-in Word List is more convenient.
Would be nice to have a button on the Built-in Word List page to add selected words to skitter.

**Comment by cschiller** [comment]
You can add words directly to Skritter by using the "s" keyboard shortcut. You don't need to add them to the word list first.

**Comment by R0leeX** [comment]
Yes, that opens a new window every time. Really annoying when reading something. 
Would prefer to add them to the world list and later on select some words I want to learn and add those to skritter.
Much more convenient and I have the option to select which world I want to add to skritter.

**Comment by cschiller** [comment]
Skritter doesn't have an API for that, so unfortunately I don't think it's possible to implement this the way you want it.

**Comment by cschiller** [comment]
Maybe you can show me in a pull request how you would implement this?

**Comment by cschiller** [comment]
Every time you add a word you will be taken to Skritter, there's no way around it. In other words, since there's not REST API for adding words it won't work the way you imagine.
